appinfo/info.xml

```diff
@@ -2,19 +2,21 @@
 <info>
```

Comment:
Replace this with:
```xml
<info xmlns:xsi= "http://www.w3.org/2001/XMLSchema-instance"
      xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
```

It helps IDEs and other tools to validate the info.xml automatically